### PR TITLE
Reverting changes on the aggregator code

### PR DESCRIPTION
**Pull request by josejulio**
- Do not parse using the action parser.
   Aggregation does not follow the same encoding rules

Reverts changes on `EmailAggregator` from https://github.com/RedHatInsights/notifications-backend/pull/595

`Hide whitespace` makes it easier to review

### CODE REVIEW

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java:61]: This is still not quite right, but it's what we had before (and similar to what we had before this PR).
We might need to collect all event types so that it's better.

**Review by gwenneg — CHANGES_REQUESTED** [review]
I haven't reviewed this yet, but don't merge it.

I need to test another PR (pod shutdown fix) before this one is merged.

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java:64]: It was failing here because our aggregation payload is not a valid `Action` nor do we need it to be a valid (at least not for now). 
It's build by the aggregation code and is not encoded to `Action`, so the payload and context are actually json objects and not json strings.

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java
Comment (gwenneg) [inline backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java:72]: Shouldn't this comment be reverted as well since the recipients no longer come from the action?

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by gwenneg** [comment]
⚠️ We're still not ready for a merge ⚠️

**Comment by josejulio** [comment]
> warning We're still not ready for a merge warning

I'm going to draft until we are ready - feel free to undraft and merge.

**Pull request merged**
